app/services/load_db.py

```diff
@@ -0,0 +1,113 @@
+from pymodm import connect
+from app.db.mongo_model_definitions import *
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+
+class LoadDb:
```

Comment:
Having service classes with `__call__` method isn't bad but the way you use it:

```
LoadDb()(data, os.getenv("MONGO_DB_TEST_ENDPOINT"))
``` 

is rather unpythonic.

I would recommend naming convention where class is an 'object' - `DBLoader` with a method `load`.

That would make calling them much prettier.

By the way - this class can also be a normal function, but if you plan to extend its features staying with class is also good.